src/models/searchDocument.js

```diff
@@ -0,0 +1,17 @@
+/* 
+* @Author: Mike Reich
+* @Date:   2016-02-21 07:04:27
+* @Last Modified 2016-02-21
+*/
+
+'use strict';
+
+import {BaseModel} from '@nxus/storage'
+
+export default BaseModel.extend({
+  identity: 'searchdocument',
+  connection: 'searcher',
+  attributes: {
+    
```

Comment:
Does this mean we're storing whatever doc structure the original models have? plus a `model` field (should that be here at a minimum?)